response_selectors/ranking_and_intent_based_response_selector/README.md

```diff
@@ -0,0 +1,21 @@
+# Ranking- and Intent-based Response Selector
+
+## Description
+
+Response Selector is a component selecting final response among the given hypotheses by different skills.
+The Ranking- and Intent-based Response Selector utilizes floating point annotations by ranking hypotheses with 
+a candidate annotator (e.g., Sentence Ranker), scaling ranking scores with heuristics depending on entities and intents,
+and finally selecting the best ranked one.
```

Comment:
Response Selector is a component that selects the final response among hypotheses provided by different skills.
The Ranking- and Intent-based Response Selector utilizes floating point annotations by ranking hypotheses with a candidate annotator (e.g., Sentence Ranker), scaling the resulting scores with heuristics based on entities and intents, and finally selecting the best ranked one.